### PR TITLE
Feature account screen Left Panel completed

### PR DESCRIPTION
**Pull request by LeonVdwRR**
In route '/AccountScreen' - Completed left side panel including search bar and contact cards 
screenshot 118

### CODE REVIEW

File: src/accountScreen/accountScreen.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+class AccountScreen extends Component {
+  
+
+
+  render() {
+    return (
+      <MuiThemeProvider>       
+        <div className='rectangle-4'>
+          <div className='searchContainer'>
+           <SearchBar hintText='Search Chats'/>
```
Comment (jaderabbit) [inline src/accountScreen/accountScreen.js]: fix indentation. perhaps install some sort of linter to check

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React, { Component } from 'react';
 import { Route, Link } from 'react-router-dom'
 import './css/index.css';
-import { headerNav } from './header.js'
+import { headerNav } from './components/headerNav'
+import AccountScreen  from './accountScreen/accountScreen'
 
 // TODO : Import all the components
```
Comment (jaderabbit) [inline src/App.js]: is this comment necessary?

File: src/accountScreen/accountScreen.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import './accountScreen.css'
+import SearchBar from 'material-ui-search-bar'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+
+
+const ChatCard = function(props) {
+  if(props.selected){
```
Comment (jaderabbit) [inline src/accountScreen/accountScreen.js]: Always put spaces around operators ( = + - * / ), and after commas:

File: src/accountScreen/accountScreen.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+.search-chats {
+    height: 23px;
+    width: 101px;
+    color: #706F6F;
+    font-family: "Avenir Next";
+    font-size: 17px;
+    line-height: 23px;
+}
+
+.searchContainer{
```
Comment (jaderabbit) [inline src/accountScreen/accountScreen.css]: I see you're switching between conventions here. 
`search-container`?

File: src/App.js [hunk 5]
```diff
@@ -1,7 +1,8 @@
 import React, { Component } from 'react';
 import { Route, Link } from 'react-router-dom'
 import './css/index.css';
-import { headerNav } from './header.js'
+import { headerNav } from './components/headerNav'
```
Comment (wscheffer) [inline src/App.js]: What is the difference between `import { headerNav } from './components/headerNav'` and `import headerNav from './components/headerNav'` would the code still work if you use `import headerNav from './components/headerNav'`?

**Comment by LeonVdwRR** [comment]
Added fixes based on Lavhe and Jade's comments. Updated branch

File: src/App.js (hunk 2 above)
Comment (LeonVdwRR) [reply src/App.js]: It was auto generated, removed

File: src/accountScreen/accountScreen.css (hunk 4 above)
Comment (LeonVdwRR) [reply src/accountScreen/accountScreen.css]: corrected conventions. Noted

File: src/accountScreen/accountScreen.js (hunk 3 above)
Comment (LeonVdwRR) [reply src/accountScreen/accountScreen.js]: Done. Noted

File: src/accountScreen/accountScreen.js (hunk 1 above)
Comment (LeonVdwRR) [reply src/accountScreen/accountScreen.js]: Indentation fixed

File: src/App.js (hunk 5 above)
Comment (LeonVdwRR) [reply src/App.js]: removed { } , used only for multiple objects from import

File: src/accountScreen/accountScreen.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+
+class AccountScreen extends Component {
+
+
+
+  render() {
+    return (
+      <div>
+        <div className='rectangle4'>
```
Comment (ReinhardtCromhout) [inline src/accountScreen/accountScreen.js]: Use proper naming conventions. Do not call css classes "rectangle4"

File: src/accountScreen/accountScreen.js [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+          avatar={props.avatar}
+        />
+        <CardText>
+          {props.text}
+        </CardText>
+      </Card>        
+    );
+  }
+
+  else {
```
Comment (ReinhardtCromhout) [inline src/accountScreen/accountScreen.js]: Do not add a new line between the closing if tag and the start of the else tag. This code is not readable

File: src/accountScreen/accountScreen.js (hunk 6 above)
Comment (LeonVdwRR) [reply src/accountScreen/accountScreen.js]: Sorry about that. The designer used dash styling for the css classes (so it was rectangle-4). I was told to use one style convention across my code, so I used camel casing (rectangle4). Is this wrong? Should I use dash casing for css specifically?

Comment (ReinhardtCromhout) [reply src/accountScreen/accountScreen.js]: My comment was not regarding naming rectangle-4 or rectangle4 (ie camel case vs dashes). My problem is that you are naming the class "rectangle4". Why is it #4. Where are 1, 2 and 3... what is the significance of the div being rectangle4? Surely there must be a more descriptive name you can give the div class

File: src/accountScreen/accountScreen.js (hunk 7 above)
Comment (LeonVdwRR) [reply src/accountScreen/accountScreen.js]: Noted. Done. Added prettier as well in latest push

File: src/accountScreen/accountScreen.js (hunk 6 above)
Comment (LeonVdwRR) [reply src/accountScreen/accountScreen.js]: Alright understood. notified team and pushed changes

**Comment by LeonVdwRR** [comment]
Addressed all requested changes

**Pull request merged**